scss/components/_banner.scss

```diff
@@ -82,4 +95,16 @@
             }
         }
     }
+
+    &__link {
+        font-size: 17px;
+
+        @include breakpoint(sm) {
+            font-size: 16px;
+        }
+    }
+
+    &__bottom-shadow {
+        box-shadow: 0 2px 0 0 #af8e0c;
```

Comment:
We should probably add this to the colours file or switch it to a similar colour (sun yellow) FBC900. Otherwise, when we go looking for what colours are used on the website we will have to dig deeper into the website/styling code.

Reply:
So we don't have this colour as one that's part of the new palette. It was a case of running `sun-yellow` through a `darken()` function - I've updated it as such in the partial